exec_test.go

```diff
@@ -118,15 +118,14 @@ func TestPutfile(t *testing.T) {
 	want := "Hello, 世界\n"
 	w := &Window{
 		body: Text{
-			file: &File{
-				b:    RuneArray(want),
-				name: filename,
-			},
+			file: NewFile(filename),
```

Comment:
Maybe `NewFile` should have a friend function that takes the backing `RuneArray` value too. Add a `TODO` please.

Also: idiomatic Go would entail calling it `MakeObservableTextBuffer(...)` (And presumably `File` will become `ObservableTextBuffer` in some set of subsequent PRs.)

Reply:
The friend function sounds like a good idea, also I have the Make function in progress on another branch, glad I had the right idea.